modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1786 @@
+RETURN relationship, 'non-unique' AS reason
+UNION
+MATCH ()-[r:REL_TYPE]-()
+WHERE r.prop IS NULL
+RETURN r AS relationship, 'non-existing' AS reason
+----
+
+|===
+
+
+[[constraint-failures-and-indexes]]
+==== Constraint failures and indexes
+
+Attempting to create any type of constraint with the same name as an existing index will fail.
+
+.Creating a node property type constraint with the same name as an existing index
+======
+
+.Create an index with the name `directors`
+[source, cypher]
+----
+CREATE INDEX directors FOR (director:Director) ON (director.name)
+----
+
+.Create a node property type constraint with the name `directors`
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT directors FOR (movie:Movie) REQUIRE movie.director IS :: STRING
+----
+
+.Error message
+[source, error]
+----
+There already exists an index called 'directors'.
+----
+
+======
+
+Creating key or property uniqueness constraints on the same schema as an existing index will fail.
+
+.Creating a node property uniqueness constraint on the same schema as an existing index
+======
+
+.Create an index for `wordCount` properties on `Book` nodes
+[source, cypher]
+----
+CREATE INDEX book_word_count FOR (book:Book) ON (book.wordCount)
+----
+
+.Create a constraint requiring all `Book` nodes to have unique `wordCount` properties
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT word_count FOR (book:Book) REQUIRE book.wordCount IS UNIQUE
+----
+
+.Error message
+[source, error]
+----
+There already exists an index (:Book {wordCount}).
+A constraint cannot be created until the index has been dropped.
+----
+
+======
+
+
+[[key-and-uniqueness-constraint-on-the-same-schema]]
+==== Creating key and property uniqueness constraints on the same schema is not allowed
+
+.Creating a node key constraint on the same schema as an existing property uniqueness constraint
+======
+
+.Create a node key constraint on the properties `title` and `publicationYear` on nodes with the `Book` label, when a property uniqueness constraint already exists on the same label and property combination
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT book_titles FOR (book:Book) REQUIRE (book.title, book.publicationYear) IS NODE KEY
+----
+
+.Error message
+[source, error]
+----
+Constraint already exists: Constraint( id=7, name='book_title_year', type='UNIQUENESS', schema=(:Book {title, publicationYear}), ownedIndex=6 )
+----
+
+======
+
+
+[[list-constraints]]
+== SHOW CONSTRAINTS
+
+To list all constraints with the default output columns, use `SHOW CONSTRAINTS`.
+If all columns are required, use `SHOW CONSTRAINTS YIELD *`.
+For the full command syntax to list constraints, see xref:constraints/syntax.adoc#list-constraints[Syntax -> SHOW CONSTRAINTS].
+
+One of the output columns from `SHOW CONSTRAINTS` is the name of the constraint.
+This can be used to drop the constraint with the xref::constraints/syntax.adoc#drop-constraint[`DROP CONSTRAINT` command].
```

Comment:
this links to te syntax section for DROP CONSTRAINT, while in the drop constraint section we link to this SHOW CONTSTRAINTS section and not the index ones, so should this link to the DROP CONSTRAINT section below instead of the syntax one to be consistent?